src/frontend/web_application/src/styles/config/theme/dark.scss

```diff
@@ -29,8 +29,16 @@ $co-color__alert:                     #9d3535;
 $co-color__alert--low:                darken($co-color__alert, 10%);
 $co-color__alert--high:               lighten($co-color__alert, 10%);
 
-$co-color__importance:                #d2bc00;
+// Privacy
 $co-color__privacy:                   #97b707;
+$co-color__importance:                #d2bc00;
+
+$privacy-palette: (
+  super: $co-color__privacy,
+  good: $co-color__importance,
```

Comment:
hmm nope. importance is another concept. 

we have `Privacy Index` (PI) and `Importance Level` (IL) which represent the importance of the message (0-100 : spam, normal, urgent)

As you spoke this morning about the palette. We started with only one color by concept (green for PI and yellow for IL). I don't know if @steventlucek gave you a palette finally.